keel-sql/src/test/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+eureka.defaultToUp=false
```

Comment:
I've consistently specified this everywhere we use Spring tests so that our scheduled stuff should never be running in the background when tests are running

Reply:
amazing